migrations/capcons/capabilities.go

```diff
@@ -28,18 +29,33 @@ import (
 type AccountCapability struct {
 	Path       interpreter.PathValue
```

Comment:
Maybe rename this to e.g. `TargetPath` to avoid this getting confused as e.g. the source path